pallets/parachain-staking/src/lib.rs

```diff
@@ -1188,16 +1188,13 @@ pub mod pallet {
 		fn compute_issuance(staked: BalanceOf<T>) -> BalanceOf<T> {
 			let config = <InflationConfig<T>>::get();
 			let round_issuance = crate::inflation::round_issuance_range::<T>(config.round);
+			// TODO: consider interpolation instead of bounded range
```

Comment:
Would you move away from the range in this case? This would affect storage, and require a migration, if so -- right?

Reply:
No, we would still use the `Range` type. The difference would be that we have different logic here in particular. 

The current logic returns `min` if `staked` is below `expect.min`, `max` if `staked` is above `expect.max` and `ideal` if in between.

Interpolation would measure the distance between `staked` and the range points to determine proportional issuance.